src/Terminal/commands/connect.ts

```diff
@@ -36,7 +36,7 @@ export function connect(
       terminal.connectToServer(player, hostname);
       return;
     }
-    terminal.error(`Cannot directly connect to ${hostname}`);
+    terminal.error(`Cannot directly connect to ${hostname}. Make sure the server is backdoored or adjacent to your current Server`);
```

Comment:
```suggestion
    terminal.error(`Cannot directly connect to ${hostname}. Make sure the server is backdoored and adjacent to your current Server`);
```

Reply:
A backdoored server doesn't have to be adjacent to your current server, so `or` would be preferred.

Reply:
Yeah, my bad. Mixed connecting and hacking in my head